### DISCUSSION TITLE
Packages/com.unity.nuget.newtonsoft-json/manual/index

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#88] [Notes]
# Packages/com.unity.nuget.newtonsoft-json/manual/index

https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@latest/index.html?subfolder=%2Fmanual%2Findex.html

**Comment by vertxxyz** [comment]
See [Unity Converters for Newtonsoft.Json](https://github.com/jilleJr/Newtonsoft.Json-for-Unity.Converters) to get converters to and from common Unity types like `Vector2`, `Vector3`, `Matrix4x4`, `Quaternion`, `Color`, even `ScriptableObject`, etc.
These types may have self referencing loops that cause them to fail serialization otherwise.
